docs/sources/guides/whats-new-in-v7-0.md

```diff
@@ -12,8 +12,161 @@ weight = -17
+
+## New tracing UI
+
+With Grafana 7.0 we are adding major support for distributed tracing. This adds an important telemetry mode to complement the existing support for metrics and logs. Traces allow you to follow how single requests travelled through a distributed system. We are starting with an integrated trace viewer and two new built-in data sources: Jaeger and Zipkin.
+
+You can use the new trace view in Explore either directly and search for a particular trace or you can configure Loki to detect trace IDs in the log lines and link directly to a trace timeline pulled from Jaeger or Zipkin data source.
+
+In the future we will add more workflows and integrations so that correlating between metrics, logs and traces is even easier.
+
+## Transformations
+
+Not just visualizing data from anywhere, in Grafana 7 you can transform it too. By chaining a simple set of point and click transformations users will be able join, pivot, filter, re-name and calculate to get the results they need. Perfect for operations across queries or data sources missing essential data transformations.
+
+Transformations and maths across queries. The data you want to visualize can come from many different places and it is usually not in exactly the right form. The new transformations feature allows you to rename fields, join separate time series together and more - data munging. Usually this requires writing code but this new feature lets you do it in the Grafana UI instead. It also lets you do maths across queries. Lots of data sources do not support this natively so being able to do it in Grafana is a killer feature. For users, with large dashboards or with heavy queries, being able to reuse the query result from one panel in another panel can be a huge performance gain.
+
+**Transformations shipping in 7.0**
+
+- **Reduce:** Reduce many rows / data points to a single value
+- **Filter by name:** Filter fields by name or regex
+- **Filter by refId:** Filter by query letter
+- **Organize fields:** Reorder, rename and hide fields.
+- **Labels to fields:** Transform time series with labels into a table where labels get's converted to fields and the result is joined by time
+- **Join by field:** Join many result sets (series) together using for example the time field. Useful for transforming time series into a table with a shared time column and where each series get it's own column.
+- **Add field from calculation:** This is a powerful transformation that allows you perform many different types of math operations and add the result as a new field. Examples:
+  - Calculate the difference between two series or fields and add the result to a new field
+  - Multiply one field with another another and add the result to a new field
+
+## Field overrides
```

Comment:
@dprokop, when we get the simple task descriptions sorted out, we should add them for transformations and field overrides. I think it will help our users understand why they should care.